test/unit/ansible_fact_parser_test.rb

```diff
@@ -0,0 +1,282 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module ForemanAnsible
+  # Checks sample Ansible facts to see if it can assign them to
+  # Host properties
+  class FactParserTest < ActiveSupport::TestCase
+    setup do
+      facts_json = HashWithIndifferentAccess.new(read_json_fixture('facts/ansible_facts.json'))
+      @facts_parser = AnsibleFactParser.new(facts_json)
+    end
```

Comment:
I'm more of a fan of using `let`and `subject`:
```suggestion
    let(:facts_json) { HashWithIndifferentAccess.new(read_json_fixture('facts/ansible_facts.json')) }
    subject { AnsibleFactParser.new(facts_json) }
```

Then in the actual test you use `subject.domain`. Maybe it's because I'm used to rspec. @tbrisker what do you think?

Reply:
I'm not the person you asked, but :-)

```
grep -r setup test | wc -l
1064

grep -r let test | wc -l  
1616
```

while it's not entirely fair comparison since `let` is used per variable, it's safe to say we don't have a single way in our code and this is a personal preference. While I also prefer `let`, I think we, as maintainers, shouldn't be asking for this and also not in the PR that moves the existing code.

Reply:
Perhaps it would be good to mention in the commit message that this moves over files from an existing repository and isn't really new code.

Reply:
I agree, also @domitea the commit shouldn't say Fixes but Refs, since the redmine issue currently tracks all other parsers too. Or create a separate RM issue per parser and use that number.

Reply:
I've updated the commit message.